app/views/record/delete/when-did-trainee-start.html

```diff
@@ -0,0 +1,20 @@
+{% extends "_templates/_record-form.html" %}
+
+{% set pageHeading %}
+  Did the trainee start their {{ "ITT" if record | courseDatesAreAmbiguous else "course" }} on time?
+{% endset %}
+
+{% set startConfirmed = true %}
+
+{% set pageHeadingTraineeName = data.record.personalDetails | getShortName %}
+{% set formAction = "./cannot-delete" %}
```

Comment:
```suggestion
{% set formAction = "./cannot-delete" %}
```

should this be `when-did-trainee-start-answer`?